packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="nutrition-facts" v-if="$page.frontmatter.nutritionItems">
+    <div
+      class="fact-item"
+      v-for="(nutrition, index) in $page.frontmatter.nutritionItems"
+      v-bind:key="index"
+    >
+      <div class="fact-item-title">{{ nutrition.title }}</div>
+      <div class="fact-bullet-list">
+        <ul>
+          <li
+            v-for="(bulletItem, index) in nutrition.bulletList"
+            v-bind:key="index"
+          >
+            {{ bulletItem }}
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+/*
+Inside index.md of the page you like put <NutritionFacts></NutritionFacts>
+And special data set for that component
+ex.
+nutritionItems:
```

Comment:
It wasn't exactly clear how to add the component to a page in the commented out example. When I added the below component to a page nothing was rendered.  Is the below syntax correct?

![image](https://user-images.githubusercontent.com/78044107/123740860-f346e580-d85d-11eb-854e-d511f7bf88b4.png)

Reply:
so not exactly, I'm going to extend an example, so this data we need to put inside a special section of an `index.md`(.md files has a special section for putting any type of data, that section wrapped by `---` sybmols) file and not to leave them inside `NutritionFacts` itself